src/ClassParser.ts

```diff
@@ -302,6 +302,15 @@ export default class ClassParser {
       if (style) return style;
     }
 
+    if (this.consumePeeked(`gap`)) {
+      style = gap(
+        this.rest,
+        this.context,
+        theme === null || theme === void 0 ? void 0 : theme.gap,
+      );
```

Comment:
we can simplify this to just: 

```suggestion
      style = gap(this.rest, this.context, theme?.gap);
```

Reply:
@rborn oh, and can you rebase? it looks like you've got some old commits in there. if you add  an upstream remote, you should be able to run `git pull --rebase upstream master` and eliminate those.